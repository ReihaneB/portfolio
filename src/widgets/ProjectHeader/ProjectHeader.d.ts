import type { Stack } from '@/widgets/MainStack/MainStack.d';
import { ImageProps } from '@/components/Image/Image';

export interface Project {
  id?: number;
  title: string;
  jobTitle: string;
  stack: Stack[];
  url: string;
  image: ImageProps;
}

export interface ProjectHeaderProps {
  project: Project;
}

export interface ProjectHeaderStoreProps {
  id: string;
}
