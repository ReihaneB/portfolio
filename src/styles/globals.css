:root {
  /* colors */
  --white: #f5f5f7;
  --cyan: #06b6d4;
  --blue: #2756df;
  --blue-lighter: #cee5ff;
  --blue-light: #dbeafe;
  --blue-dark: #1e40af;
  --focus-blue: #93c5fd;
  --grey: #6b7280;
  --grey-light: #e5e7eb;
}

* {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
  border: 0;
  font: inherit;
  vertical-align: baseline;
}

html,
body {
  max-width: 100vw;
  overflow-x: hidden;
}

details,
figcaption,
figure,
footer,
hgroup,
menu,
nav {
  display: block,
}

ol,
ul {
  list-style: none;
}

blockquote,
q {
  quotes: none;
}

blockquote::before,
blockquote::after,
q::before,
q::after {
  content: '';
  content: none;
}

table {
  border-collapse: collapse;
  border-spacing: 0;
}

a {
  color: inherit;
  text-decoration: none;
}

h1 {
  font-size: 30px;
  line-height: 38px;
  font-weight: bold;
}

h2 {
  font-size: 20px;
  line-height: 24px;
  font-weight: bold;
}

h3 {
  font-size: 16px;
  line-height: 20px;
  font-weight: bold;
}

h4 {
  font-size: 14px;
  line-height: 18px;
}

h5 {
  font-size: 12px;
  line-height: 16px;
}

p {
  font-size: 16px;
  line-height: 24px;
}