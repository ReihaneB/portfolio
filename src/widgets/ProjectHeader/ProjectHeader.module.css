.root {
  display: grid;
  gap: 16px;
  grid-template-columns: auto 1fr;
  grid-template-areas:
    "title title"
    "stack link"
    "image image";
  
  @media (width <= 768px) {
    grid-template-columns: 1fr;
    grid-template-areas:
      "title"
      "stack"
      "image"
      "link";
  }
}

.titleContainer {
  grid-area: title;
}

.stackContainer {
  grid-area: stack;
}

.linkContainer {
  grid-area: link;

  white-space: nowrap;
  justify-self: end;

  @media (width <= 768px) {
    justify-self: center;
  }
}

.imageContainer {
  grid-area: image;
}

.image {
  width: 100%;
  height: auto;
  border-radius: 16px;
}
