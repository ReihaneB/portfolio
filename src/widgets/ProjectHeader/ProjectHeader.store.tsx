import { getProjectHeader } from '@/api/getProjectHeader';

import ProjectHeader from './ProjectHeader';

import { ProjectHeaderStoreProps } from './ProjectHeader.d';

async function ProjectHeaderStore({ id }: ProjectHeaderStoreProps) {
  const project = await getProjectHeader({ id });

  return (
    <ProjectHeader
      project={project}
    />
  );
}

export default ProjectHeaderStore;
