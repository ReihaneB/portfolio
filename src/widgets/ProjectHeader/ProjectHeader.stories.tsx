import type { Meta, StoryObj } from '@storybook/react';

import ProjectHeader from './ProjectHeader';

import styles from './ProjectHeader.stories.module.css';

const meta: Meta<typeof ProjectHeader> = {
  title: 'Widgets/ProjectHeader',
  component: ProjectHeader,
  decorators: [
    Story => (
      <div className={styles.root}>
        <Story />
      </div>
    ),
  ],
};

export default meta;
type Story = StoryObj<typeof ProjectHeader>;

const defaultArgs = {
  project: {
    id: 1,
    title: 'Project Title',
    jobTitle: 'Job Title',
    url: '/',
    stack: [{
      id: 1,
      name: 'React',
    },
    {
      id: 2,
      name: 'TypeScript',
    },
    {
      id: 3,
      name: 'Next.js',
    },
    {
      id: 4,
      name: 'Tailwind CSS',
    },
    ],
    image: {
      url: 'https://via.placeholder.com/1920x1080',
      alt: 'Project Image',
    },
  },
};

export const Playground: Story = {
  args: {
    ...defaultArgs,
  },
};

export const SquareImage: Story = {
  args: {
    ...defaultArgs,
    project: {
      ...defaultArgs.project,
      image: {
        url: 'https://via.placeholder.com/500x500',
        alt: 'Project Image',
      },
    },
  },
};

export const _16x9Image: Story = {
  args: {
    ...defaultArgs,
    project: {
      ...defaultArgs.project,
      image: {
        url: 'https://via.placeholder.com/1920x1080',
        alt: 'Project Image',
      },
    },
  },
};

export const _4x3Image: Story = {
  args: {
    ...defaultArgs,
    project: {
      ...defaultArgs.project,
      image: {
        url: 'https://via.placeholder.com/800x600',
        alt: 'Project Image',
      },
    },
  },
};

export const LongTitle: Story = {
  args: {
    ...defaultArgs,
    project: {
      ...defaultArgs.project,
      title: 'This is a very long title that should not be truncated',
    },
  },
};

export const LongJobTitle: Story = {
  args: {
    ...defaultArgs,
    project: {
      ...defaultArgs.project,
      jobTitle: 'This is a very long job title that should not be truncated',
    },
  },
};

export const LongStackName: Story = {
  args: {
    ...defaultArgs,
    project: {
      ...defaultArgs.project,
      stack: [
        {
          id: 1,
          name: 'React',
        },
        {
          id: 2,
          name: 'This is a second very long stack name that should not be truncated',
        },
        {
          id: 3,
          name: 'Next.js',
        },
        {
          id: 4,
          name: 'This is a fourth very long stack name that should not be truncated',
        },
      ],
    },
  },
};

export const LongData: Story = {
  args: {
    ...defaultArgs,
    project: {
      ...defaultArgs.project,
      title: 'This is a very long title that should not be truncated',
      jobTitle: 'This is a very long job title that should not be truncated',
      stack: [
        {
          id: 1,
          name: 'React',
        },
        {
          id: 2,
          name: 'This is a second very long stack name that should not be truncated',
        },
        {
          id: 3,
          name: 'Next.js',
        },
        {
          id: 4,
          name: 'This is a fourth very long stack name that should not be truncated',
        },
      ],
    },
  },
};
