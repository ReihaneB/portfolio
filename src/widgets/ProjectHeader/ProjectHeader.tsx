import Image from 'next/image';
import NextLink from 'next/link';

import Link from '@/components/Link/Link';

import MainStack from '../MainStack/MainStack';

import styles from './ProjectHeader.module.css';
import type { ProjectHeaderProps } from './ProjectHeader.d';

function ProjectHeader({
  project,
  ...rest
}: ProjectHeaderProps) {
  return (
    <div
      className={styles.root}
      {...rest}
    >
      <div className={styles.titleContainer}>
        <h2>{project.title}</h2>
        <p>{project.jobTitle}</p>
      </div>
      <div className={styles.stackContainer}>
        <MainStack stack={project.stack} />
      </div>
      <div className={styles.linkContainer}>
        <Link
          to={project.url}
          color="secondary"
        >
          Voir le projet
        </Link>
      </div>
      <NextLink
        href={project.url}
        passHref
        className={styles.imageContainer}
      >
        <Image
          src={project.image.url}
          className={styles.image}
          alt={project.image.alt}
          width={0}
          height={0}
          priority
        />
      </NextLink>
    </div>
  );
}

export default ProjectHeader;
