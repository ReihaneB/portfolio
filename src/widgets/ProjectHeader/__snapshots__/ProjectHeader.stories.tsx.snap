// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Widgets/ProjectHeader _4x3Image smoke-test 1`] = `
<main class="inter-normal">
  <div class="src-widgets-ProjectHeader-ProjectHeader-stories-module__root--hHMpH">
    <div class="src-widgets-ProjectHeader-ProjectHeader-module__root--ZnpQA">
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__titleContainer--eg7T9">
        <h2>
          Project Title
        </h2>
        <p>
          Job Title
        </p>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__stackContainer--WS2tI">
        <ul class="src-widgets-MainStack-MainStack-module__root--dscGt">
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              React
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              TypeScript
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Next.js
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Tailwind CSS
            </div>
          </li>
        </ul>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__linkContainer--Xyp1T">
        <a class="src-components-Link-Link-module__root--ggyqZ src-components-Link-Link-module__secondary--tiYlA"
           data-testid="link-component"
           href="/"
        >
          Voir le projet
        </a>
      </div>
      <a class="src-widgets-ProjectHeader-ProjectHeader-module__imageContainer--hhsDI"
         href="/"
      >
        <img alt="Project Image"
             fetchpriority="high"
             width="0"
             height="0"
             decoding="async"
             data-nimg="1"
             class="src-widgets-ProjectHeader-ProjectHeader-module__image--BDZ5_"
             srcset="https://via.placeholder.com/800x600?w=16&amp;q=75 1x"
             src="https://via.placeholder.com/800x600?w=16&amp;q=75"
             style="color: transparent;"
        >
      </a>
    </div>
  </div>
</main>
`;

exports[`Widgets/ProjectHeader _16x9Image smoke-test 1`] = `
<main class="inter-normal">
  <div class="src-widgets-ProjectHeader-ProjectHeader-stories-module__root--hHMpH">
    <div class="src-widgets-ProjectHeader-ProjectHeader-module__root--ZnpQA">
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__titleContainer--eg7T9">
        <h2>
          Project Title
        </h2>
        <p>
          Job Title
        </p>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__stackContainer--WS2tI">
        <ul class="src-widgets-MainStack-MainStack-module__root--dscGt">
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              React
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              TypeScript
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Next.js
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Tailwind CSS
            </div>
          </li>
        </ul>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__linkContainer--Xyp1T">
        <a class="src-components-Link-Link-module__root--ggyqZ src-components-Link-Link-module__secondary--tiYlA"
           data-testid="link-component"
           href="/"
        >
          Voir le projet
        </a>
      </div>
      <a class="src-widgets-ProjectHeader-ProjectHeader-module__imageContainer--hhsDI"
         href="/"
      >
        <img alt="Project Image"
             fetchpriority="high"
             width="0"
             height="0"
             decoding="async"
             data-nimg="1"
             class="src-widgets-ProjectHeader-ProjectHeader-module__image--BDZ5_"
             srcset="https://via.placeholder.com/1920x1080?w=16&amp;q=75 1x"
             src="https://via.placeholder.com/1920x1080?w=16&amp;q=75"
             style="color: transparent;"
        >
      </a>
    </div>
  </div>
</main>
`;

exports[`Widgets/ProjectHeader LongData smoke-test 1`] = `
<main class="inter-normal">
  <div class="src-widgets-ProjectHeader-ProjectHeader-stories-module__root--hHMpH">
    <div class="src-widgets-ProjectHeader-ProjectHeader-module__root--ZnpQA">
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__titleContainer--eg7T9">
        <h2>
          This is a very long title that should not be truncated
        </h2>
        <p>
          This is a very long job title that should not be truncated
        </p>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__stackContainer--WS2tI">
        <ul class="src-widgets-MainStack-MainStack-module__root--dscGt">
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              React
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              This is a second very long stack name that should not be truncated
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Next.js
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              This is a fourth very long stack name that should not be truncated
            </div>
          </li>
        </ul>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__linkContainer--Xyp1T">
        <a class="src-components-Link-Link-module__root--ggyqZ src-components-Link-Link-module__secondary--tiYlA"
           data-testid="link-component"
           href="/"
        >
          Voir le projet
        </a>
      </div>
      <a class="src-widgets-ProjectHeader-ProjectHeader-module__imageContainer--hhsDI"
         href="/"
      >
        <img alt="Project Image"
             fetchpriority="high"
             width="0"
             height="0"
             decoding="async"
             data-nimg="1"
             class="src-widgets-ProjectHeader-ProjectHeader-module__image--BDZ5_"
             srcset="https://via.placeholder.com/1920x1080?w=16&amp;q=75 1x"
             src="https://via.placeholder.com/1920x1080?w=16&amp;q=75"
             style="color: transparent;"
        >
      </a>
    </div>
  </div>
</main>
`;

exports[`Widgets/ProjectHeader LongJobTitle smoke-test 1`] = `
<main class="inter-normal">
  <div class="src-widgets-ProjectHeader-ProjectHeader-stories-module__root--hHMpH">
    <div class="src-widgets-ProjectHeader-ProjectHeader-module__root--ZnpQA">
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__titleContainer--eg7T9">
        <h2>
          Project Title
        </h2>
        <p>
          This is a very long job title that should not be truncated
        </p>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__stackContainer--WS2tI">
        <ul class="src-widgets-MainStack-MainStack-module__root--dscGt">
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              React
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              TypeScript
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Next.js
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Tailwind CSS
            </div>
          </li>
        </ul>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__linkContainer--Xyp1T">
        <a class="src-components-Link-Link-module__root--ggyqZ src-components-Link-Link-module__secondary--tiYlA"
           data-testid="link-component"
           href="/"
        >
          Voir le projet
        </a>
      </div>
      <a class="src-widgets-ProjectHeader-ProjectHeader-module__imageContainer--hhsDI"
         href="/"
      >
        <img alt="Project Image"
             fetchpriority="high"
             width="0"
             height="0"
             decoding="async"
             data-nimg="1"
             class="src-widgets-ProjectHeader-ProjectHeader-module__image--BDZ5_"
             srcset="https://via.placeholder.com/1920x1080?w=16&amp;q=75 1x"
             src="https://via.placeholder.com/1920x1080?w=16&amp;q=75"
             style="color: transparent;"
        >
      </a>
    </div>
  </div>
</main>
`;

exports[`Widgets/ProjectHeader LongStackName smoke-test 1`] = `
<main class="inter-normal">
  <div class="src-widgets-ProjectHeader-ProjectHeader-stories-module__root--hHMpH">
    <div class="src-widgets-ProjectHeader-ProjectHeader-module__root--ZnpQA">
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__titleContainer--eg7T9">
        <h2>
          Project Title
        </h2>
        <p>
          Job Title
        </p>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__stackContainer--WS2tI">
        <ul class="src-widgets-MainStack-MainStack-module__root--dscGt">
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              React
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              This is a second very long stack name that should not be truncated
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Next.js
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              This is a fourth very long stack name that should not be truncated
            </div>
          </li>
        </ul>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__linkContainer--Xyp1T">
        <a class="src-components-Link-Link-module__root--ggyqZ src-components-Link-Link-module__secondary--tiYlA"
           data-testid="link-component"
           href="/"
        >
          Voir le projet
        </a>
      </div>
      <a class="src-widgets-ProjectHeader-ProjectHeader-module__imageContainer--hhsDI"
         href="/"
      >
        <img alt="Project Image"
             fetchpriority="high"
             width="0"
             height="0"
             decoding="async"
             data-nimg="1"
             class="src-widgets-ProjectHeader-ProjectHeader-module__image--BDZ5_"
             srcset="https://via.placeholder.com/1920x1080?w=16&amp;q=75 1x"
             src="https://via.placeholder.com/1920x1080?w=16&amp;q=75"
             style="color: transparent;"
        >
      </a>
    </div>
  </div>
</main>
`;

exports[`Widgets/ProjectHeader LongTitle smoke-test 1`] = `
<main class="inter-normal">
  <div class="src-widgets-ProjectHeader-ProjectHeader-stories-module__root--hHMpH">
    <div class="src-widgets-ProjectHeader-ProjectHeader-module__root--ZnpQA">
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__titleContainer--eg7T9">
        <h2>
          This is a very long title that should not be truncated
        </h2>
        <p>
          Job Title
        </p>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__stackContainer--WS2tI">
        <ul class="src-widgets-MainStack-MainStack-module__root--dscGt">
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              React
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              TypeScript
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Next.js
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Tailwind CSS
            </div>
          </li>
        </ul>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__linkContainer--Xyp1T">
        <a class="src-components-Link-Link-module__root--ggyqZ src-components-Link-Link-module__secondary--tiYlA"
           data-testid="link-component"
           href="/"
        >
          Voir le projet
        </a>
      </div>
      <a class="src-widgets-ProjectHeader-ProjectHeader-module__imageContainer--hhsDI"
         href="/"
      >
        <img alt="Project Image"
             fetchpriority="high"
             width="0"
             height="0"
             decoding="async"
             data-nimg="1"
             class="src-widgets-ProjectHeader-ProjectHeader-module__image--BDZ5_"
             srcset="https://via.placeholder.com/1920x1080?w=16&amp;q=75 1x"
             src="https://via.placeholder.com/1920x1080?w=16&amp;q=75"
             style="color: transparent;"
        >
      </a>
    </div>
  </div>
</main>
`;

exports[`Widgets/ProjectHeader Playground smoke-test 1`] = `
<main class="inter-normal">
  <div class="src-widgets-ProjectHeader-ProjectHeader-stories-module__root--hHMpH">
    <div class="src-widgets-ProjectHeader-ProjectHeader-module__root--ZnpQA">
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__titleContainer--eg7T9">
        <h2>
          Project Title
        </h2>
        <p>
          Job Title
        </p>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__stackContainer--WS2tI">
        <ul class="src-widgets-MainStack-MainStack-module__root--dscGt">
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              React
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              TypeScript
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Next.js
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Tailwind CSS
            </div>
          </li>
        </ul>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__linkContainer--Xyp1T">
        <a class="src-components-Link-Link-module__root--ggyqZ src-components-Link-Link-module__secondary--tiYlA"
           data-testid="link-component"
           href="/"
        >
          Voir le projet
        </a>
      </div>
      <a class="src-widgets-ProjectHeader-ProjectHeader-module__imageContainer--hhsDI"
         href="/"
      >
        <img alt="Project Image"
             fetchpriority="high"
             width="0"
             height="0"
             decoding="async"
             data-nimg="1"
             class="src-widgets-ProjectHeader-ProjectHeader-module__image--BDZ5_"
             srcset="https://via.placeholder.com/1920x1080?w=16&amp;q=75 1x"
             src="https://via.placeholder.com/1920x1080?w=16&amp;q=75"
             style="color: transparent;"
        >
      </a>
    </div>
  </div>
</main>
`;

exports[`Widgets/ProjectHeader SquareImage smoke-test 1`] = `
<main class="inter-normal">
  <div class="src-widgets-ProjectHeader-ProjectHeader-stories-module__root--hHMpH">
    <div class="src-widgets-ProjectHeader-ProjectHeader-module__root--ZnpQA">
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__titleContainer--eg7T9">
        <h2>
          Project Title
        </h2>
        <p>
          Job Title
        </p>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__stackContainer--WS2tI">
        <ul class="src-widgets-MainStack-MainStack-module__root--dscGt">
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              React
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              TypeScript
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Next.js
            </div>
          </li>
          <li>
            <div class="src-components-Tag-Tag-module__rootStyle--Cd2YJ"
                 data-testid="tag-component"
                 tabindex="-1"
            >
              Tailwind CSS
            </div>
          </li>
        </ul>
      </div>
      <div class="src-widgets-ProjectHeader-ProjectHeader-module__linkContainer--Xyp1T">
        <a class="src-components-Link-Link-module__root--ggyqZ src-components-Link-Link-module__secondary--tiYlA"
           data-testid="link-component"
           href="/"
        >
          Voir le projet
        </a>
      </div>
      <a class="src-widgets-ProjectHeader-ProjectHeader-module__imageContainer--hhsDI"
         href="/"
      >
        <img alt="Project Image"
             fetchpriority="high"
             width="0"
             height="0"
             decoding="async"
             data-nimg="1"
             class="src-widgets-ProjectHeader-ProjectHeader-module__image--BDZ5_"
             srcset="https://via.placeholder.com/500x500?w=16&amp;q=75 1x"
             src="https://via.placeholder.com/500x500?w=16&amp;q=75"
             style="color: transparent;"
        >
      </a>
    </div>
  </div>
</main>
`;
